From 56592c49fd9cb6ee2934d0d2c2c217ebafa1ded8 Mon Sep 17 00:00:00 2001
From: Takashi Sakamoto <o-takashi@sakamocchi.jp>
Date: Sun, 17 Sep 2023 11:52:28 +0900
Subject: [PATCH] hwdb: ieee1394-unit-function: remove superfluous Weiss
 Engineering DAC1 entry

Michele Perrone and Rolf Anderegg reported that their product with
model_id 0x000005 was experimental for internal use and never for
consumer market.

* https://lore.kernel.org/alsa-devel/a5b63c37-7f6f-46f3-a87f-e4e7c0c9b5c8@weiss.ch/

This commit deletes corresponding entry.

(cherry picked from commit 7760d8de3d44c0b6ffe9bd39460388fab4d1b2b9)

Related: RHEL-5950
---
 hwdb.d/80-ieee1394-unit-function.hwdb | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/hwdb.d/80-ieee1394-unit-function.hwdb b/hwdb.d/80-ieee1394-unit-function.hwdb
index cf55f5524c..33bf7d93d8 100644
--- a/hwdb.d/80-ieee1394-unit-function.hwdb
+++ b/hwdb.d/80-ieee1394-unit-function.hwdb
@@ -912,12 +912,6 @@ ieee1394:ven00001C6Amo00000004sp00001C6Aver00000001
  ID_MODEL_FROM_DATABASE=AFI1
  IEEE1394_UNIT_FUNCTION_AUDIO=1
 
-ieee1394:node:ven0x001c6amo0x000005units0x001c6a:0x000001
-ieee1394:ven00001C6Amo00000005sp00001C6Aver00000001
- ID_VENDOR_FROM_DATABASE=Weiss Engineering
- ID_MODEL_FROM_DATABASE=DAC1
- IEEE1394_UNIT_FUNCTION_AUDIO=1
-
 ieee1394:node:ven0x001c6amo0x000006units0x001c6a:0x000001
 ieee1394:ven00001C6Amo00000006sp00001C6Aver00000001
  ID_VENDOR_FROM_DATABASE=Weiss Engineering
